.github/workflows/main.yml

```diff
@@ -501,8 +501,8 @@ jobs:
         # We don't have external services in Helm tests, so we ignore corresponding cases
         # They are still tested without Helm
         run: |
-          kubectl cp tests/mounted_file_share/images $(kubectl get pods -l component=server -o jsonpath='{.items[0].metadata.name}'):/home/django/share
-          kubectl cp tests/mounted_file_share/videos $(kubectl get pods -l component=server -o jsonpath='{.items[0].metadata.name}'):/home/django/share
+          kubectl cp tests/mounted_file_share $(kubectl get pods -l component=server -o jsonpath='{.items[0].metadata.name}'):/home/django/share/
+          kubectl exec $(kubectl get pods -l component=server -o jsonpath='{.items[0].metadata.name}') -- bash -c "mv /home/django/share/mounted_file_share/* /home/django/share/ && rm -r /home/django/share/mounted_file_share"
```

Comment:
Couldn't this be simplified to 

```
kubectl cp tests/mounted_file_share/* $(kubectl get pods -l component=server -o jsonpath='{.items[0].metadata.name}'):/home/django/share/
```

?

Reply:
The official example 2 says to use `kubectl exec` and the reasons are apparently discussed [here](https://devops.stackexchange.com/questions/6669/copy-multiple-files-using-wildcard-from-kubernetes-container). I didn't try to dig this quesiton deeper.

Reply:
Both the example 2 and the question you link deal with downloading files _from_ a pod. Here we're uploading files _to_ a pod, which is much easier, since the shell will expand the wildcard.

Reply:
Ah, okay, I see that `kubectl cp` doesn't actually support more than one source. :slightly_frowning_face:

How about this then?

```
tar -C tests/mounted_file_share/ -c . | \
    kubectl exec $(kubectl get pods -l component=server -o jsonpath='{.items[0].metadata.name}') -- tar -C /home/django/share -x
```